qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/ClusteringRule.java

```diff
@@ -630,7 +630,7 @@ private void waitForNewLeaderOfPartitions(
   public void waitForTopology(final Predicate<List<BrokerInfo>> topologyPredicate) {
     Awaitility.await()
         .pollInterval(Duration.ofMillis(100))
-        .atMost(Duration.ofSeconds(10))
+        .atMost(Duration.ofSeconds(60))
```

Comment:
In some cases, this will increase the runtime of our ci pipelines. I have a feeling that many tests use this method, increasing the chance of increased duration of ci pipelines. Did you also consider alternative solutions?

Reply:
This would increase the runtime only if the leader election was not fast. In that case, with 10 seconds, the tests would fail. I can't think of alternative solutions - there is no upperbound on how long the raft leader election last.

Reply:
As discussed, if it took more than 10 seconds the test would fail, now it might take a bit longer but still have the opportunity to succeed.